### PR TITLE
Add: BigLake Iceberg Table Examples

### PR DESCRIPTION
**Pull request by nevzheng**
## Description

Fixes #<ISSUE-NUMBER>

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [x] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [x] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [x] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [x] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [x] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [x] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s): https://cloud.google.com/bigquery/docs/iceberg-tables

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [ ] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [ ] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

**Comment by nevzheng** [comment]
I've mailed https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1773 to fix

**Comment by nevzheng** [comment]
This CL is dependent on the connection API which I added in go/tf-samples/pull/470, which should be merged prior to re-running this build.

File: bigquery/biglake/bigquery_create_iceberg_table_blms/main.tf [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+/**
+* This sample creates a BigLake Iceberg Table using a BigLake Metastore URI.
+* https://cloud.google.com/bigquery/docs/iceberg-tables#create-using-biglake-metastore
+*/
+
+# [START bigquery_create_iceberg_blms]
+
+
+# This queries the provider for project information.
+data "google_project" "project" {}
```
Comment (glasnt) [inline bigquery/biglake/bigquery_create_iceberg_table_blms/main.tf:26]: tflint is failing because this has been declared but not referenced

**Comment by nevzheng** [comment]
I'm focusing on other priorities at the moment.

**Pull request closed without merging**